patroni/dcs/kubernetes.py

```diff
@@ -161,30 +166,32 @@ def headers(self):
 
 class K8sObject(object):
 
-    def __init__(self, kwargs):
+    def __init__(self, kwargs: Dict[str, Any]):
         self._dict = {k: self._wrap(k, v) for k, v in kwargs.items()}
 
-    def get(self, name, default=None):
+    def get(self, name: str, default: Optional[Any] = None) -> Any:
```

Comment:
```suggestion
    def get(self, name: str, default: Optional[Any] = None) -> Optional[Any]:
```

Should we make it explicit that `None` is a valid return value?

Reply:
This one doesn't create problems.